kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,12 +1,36 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018,2020 Oracle Corporation and/or its affiliates.  All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 #  This script is to create or delete Ingress controllers. We support two ingress controllers: traefik and voyager.
 
 MYDIR="$(dirname "$(readlink -f "$0")")"
 VNAME=voyager-operator  # release name of Voyager
 TNAME=traefik-operator  # release name of Traefik
+VSPACE=voyager # NameSpace for Voyager
+TSPACE=traefik   # NameSpace for Traefik
+
+helm version --short | grep v2
```

Comment:
check specifically for the string you want instead of assuming that the lack of presence means it is the other version

Reply:
this could also fail if the server version string has v2 or v3 in it.

Reply:
add `--client`

Reply:
@markxnelson 

> "check specifically for the string you want instead of assuming that the lack of presence means it is the other version".   

The code assumes v3 or later if v2 is not present so that future version updates beyond v3 don't necessarily break the code.   What do you advise?   Fail if it's not v2 or v3?

Reply:
From V3 and above there is no tiller deployment. 

[2.x]  helm version --short --client
Client: v2.16.1+gbbdfe5e
[3.x]  helm version --short --client
v3.0.2+g19e47ee

I have added --client parameter to verify the client only version  as follows ...
helm version --short --client | grep v2